populus/chain/base.py

```diff
@@ -173,7 +173,7 @@ def contract_factories(self):
             "the `chain.provider` and `chain.provider` API to access contract "
             "factory data"
         ))
-        compiled_contracts = self.project.compiled_contracts
+        compiled_contracts = self.project.compiled_contracts['contracts']
```

Comment:
this is fixed in #293 but I believe this should also be `project.compiled_contract_data`